api/transformers.py

```diff
@@ -143,11 +143,6 @@ def format(self, value):
     ('xp', fields.Integer),
 ])
 
-player_stats_list = player_stats.copy()
-player_stats_list.update(OrderedDict([
-    ('position', fields.Integer(attribute='leaderboard_position')),
```

Comment:
Check what I have done in the past and make sure corresponding API blueprints are in sync